src/main/java/org/zalando/nakadi/service/subscription/zk/AbstractZkSubscriptionClient.java

```diff
@@ -350,9 +350,18 @@ public final ZKSubscription subscribeForTopologyChanges(final Runnable onTopolog
     }
 
     @Override
-    public final ZkSubscriptionNode getZkSubscriptionNodeLocked() {
+    public final ZkSubscriptionNode getZkSubscriptionNodeLocked() throws SubscriptionNotInitializedException {
```

Comment:
I do not see the method throwing SubscriptionNotInitializedException instead it does throw NakadiRuntimeException